### ISSUE TITLE
Commented out code to keep track of pokemon in team doesn't work.

### ISSUE THREAD

**Issue by bkinaan** [bkinaan/software-saturdays-final-project#1]
Uncomment the code in the handle button press function in PokemonTeam to reproduce the error.

Steps to reproduce:
1. Add a pokemon to the team
2. Remove a pokemon from the team
3. Add any pokemon to the team -- results in an error

The error says that .includes() does not exist.

**Comment by bkinaan** [comment]
From Bing:
`Array(0)` and `[]` are equivalent and both represent an empty array. If the `list` state variable is updated with an empty array after calling the code I provided, it means that all items in the list have been removed.

If you are still getting an error when calling the `includes` method on the `list` variable after it has been updated with an empty array, there might be other issues causing the error.
